src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+function getNumberFromSalary(item) {
+  const result = +[...item.dataset.salary].map((char) => {
+    if (isNaN(char)) {
+      return;
+    }
+
+    return char;
+  }).join('');
+
+  return result;
+}
+
+function getEmployees(arr) {
+  const result = [];
+
+  arr.forEach((el) => {
+    result.push({
+      name: el.innerText,
+      position: el.dataset.position,
+      salary: el.dataset.salary,
+      age: el.dataset.age,
+    });
+  });
+
+  return result;
+}
+
+const list = document.querySelector('UL');
+const employees = [...document.getElementsByTagName('li')];
+
+employees.sort((a, b) => {
+  if (getNumberFromSalary(a) - getNumberFromSalary(b) > 0) {
+    return -1;
+  }
```

Comment:
```suggestion
  return getNumberFromSalary(a) - getNumberFromSalary(b)
```

Reply:
if use your code, we must reverse array, i don`t want +1 unneeded step, so you need talk to me why your is best practice.